pkg/translator/api/constants.go

```diff
@@ -0,0 +1,5 @@
+package api
+
+var IngressEdgeLBHTTPModeDisabled = "disabled"
```

Comment:
nit: `const`s rather than `var`s, and grouped in one `const (` block.

Reply:
Done.